.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule "src/e2eTest/resources/nexus-publish-e2e-multi-project"]
 	path = src/e2eTest/resources/nexus-publish-e2e-multi-project
 	url = git@github.com:gradle-nexus/nexus-publish-e2e-multi-project.git
+[submodule "src/e2eTest/resources/nexus-publish-e2e-minimal-ivy"]
+	path = src/e2eTest/resources/nexus-publish-e2e-minimal-ivy
+	url = git@github.com:maczikasz/nexus-publish-e2e-minimal-ivy.git
+[submodule "src/e2eTest/resources/nexus-publish-e2e-minimal-ivy-sbt"]
+	path = src/e2eTest/resources/nexus-publish-e2e-minimal-ivy-sbt
+	url = git@github.com:maczikasz/nexus-publish-e2e-minimal-ivy-sbt.git
```

Comment:
We should move these to the `gradle-nexus` org but that can be done in a separate PR.

Reply:
Some contributors have been asking about moving the e2e projects into the main repo. What do you think about giving it a try with the new ones?

It should be easier for people to contribute to those e2e projects. The downside is the fact of putting extra stuff (such as wrapper.jar) into the project and probably more problematic dependency bumping with Renovate (although still possible).

Reply:
@maczikasz Could you try to move those two projects directly into the `src/e2eTest/resources`?

Reply:
Btw, what do you think (could be in a separate PR, if preferred) about putting those two new projects in the one directory with two different set of `build.gradle` and `settings.gradle`, with both `build.gradle` including some `e2e-publishing.gradle` (also in that same directory) containing the "e2e releasing related 'magic'"?

Unfortunately `-b` and `-c` seems to be deprecated in 8.x:
> -b, --build-file                   Specify the build file. [deprecated]

What does Gradle Test Kit offers as a replacement?

Reply:
I think they need to be in two different directories but could apply a common `e2e-publishing.gradle` script from a shared directory.

Reply:
For functional tests `-b` and `-c` can be very handy. Why originally they were deprecated?